ansible/roles/package-upgrade/tasks/partials/brew.yml

```diff
@@ -36,10 +36,20 @@
     script: files/install-homebrew.sh
     when: os == "macos11.0" and not armbrew.stat.exists
 
+  # Ignoring deprecation errors in MacOS 10.x
+  - name: Upgrade installed packages 
+    become_user: "{{ ansible_user }}"
+    homebrew:
+      upgrade_all: yes
+    when: os != "macos11.0"
+    ignore_errors: true
+
+
   - name: Upgrade installed packages
     become_user: "{{ ansible_user }}"
     homebrew:
       upgrade_all: yes
+    when: os == "macos11.0"    
```

Comment:
Can we make the condition less specific to avoid having to change it when we add macOS 12 and 13 hosts?

Reply:
Great suggestion @targos! I fix in [1ea3173](https://github.com/nodejs/build/pull/3130/commits/1ea31738e0a8df10008eb13ac53de416c1e662fc)